modules/publication/jsx/projectFields.js

```diff
@@ -39,7 +39,7 @@ class EmailElement extends React.Component {
         </label>
         <div className="col-sm-7">
           <input
-            type="text"
+            type="email"
```

Comment:
i'm a bit confused here. do you also want to remove this EmailElement component out of this jsx file?

Reply:
@zaliqarosli I think I left this because the component has different column than the other component.